website/docs/index.html.markdown

```diff
@@ -11,6 +11,8 @@ The Azure DevOps provider can be used to configure Azure DevOps project in [Micr
 
 Use the navigation to the left to read about the available resources.
 
+Interested in the provider's latest features, or want to make sure you're up to date? Check out the [changelog](https://github.com/microsoft/terraform-provider-azuredevops/blob/master/CHANGELOG.md) for version information and release notes.
```

Comment:
Instead of linking the url, could you link the docs location in the repo? Like: `[changelog](../../CHANGELOG.md)`

Reply:
I definitely can. I will say I modeled this off. If you still want me to change things, let me know and I'll do the update.
https://www.terraform.io/docs/providers/azurerm/index.html
https://www.terraform.io/docs/providers/azuread/index.html

Reply:
I definitely can, but I will say I modeled this off of these links. If you still want me to move ahead, I'll be happy to quickly update it.
https://www.terraform.io/docs/providers/azurerm/index.html
https://www.terraform.io/docs/providers/azuread/index.html

Reply:
Looking at that, while they use an url for the change log, they use doc reference for the other ones. 
![image](https://user-images.githubusercontent.com/12761692/87427980-89481880-c5d9-11ea-8fd0-02da80d4588d.png)
However the other ones are linked in the website. Hmm, doing a quick check with inspect, I think the url would be broken if we used my suggestion.

**Conclusion**:
You're correct! Thank you so much for your contribution :) I'll merge it.